core/state/snapshot/snapshot.go

```diff
@@ -169,11 +169,11 @@ type Tree struct {
 // store (with a number of memory layers from a journal), ensuring that the head
 // of the snapshot matches the expected one.
 //
-// If the snapshot is missing or the disk layer is broken, the entire is deleted
+// If the snapshot is missing or the disk layer is broken, the entire snapshot is deleted
 // and will be reconstructed from scratch based on the tries in the key-value
 // store, on a background thread. If the memory layers from the journal is not
 // continuous with disk layer or the journal is missing, all diffs will be discarded
-// iff it's in "recovery" mode, otherwise rebuild is mandatory.
+// if it's in "recovery" mode, otherwise rebuild is mandatory.
```

Comment:
`iff` usually means `if and only if`. Not sure if that's the case here, but probably. 

Funny, because it's the second time in a short span of time that someone has tried to fix this word :)

Reply:
you learn something new every day. thanks! 
iff seems right. amended.